### ISSUE TITLE
Which license the project uses is unclear. Proposal to license as GPLv2 or later

### ISSUE THREAD

**Issue by samcv** [brain-workshop/brainworkshop#37]
The project's code is nominally under GPL licensing as stated in brainworkshop.py

```
 Copyright (C) 2009-2011: Paul Hoskinson (plhosk@gmail.com)
#
# The code is GPL licensed (https://www.gnu.org/copyleft/gpl.html)
```

The GPL page linked says that if no version is specified, you should interpret that any any GPL version you wish. I believe we should standardize on "GPLv2 or later". 

Reasons for using GPLv2 or later and not GPLv3: GPLv3's text is twice as long and more complicated. I don't think the GPLv3 additions improve this project specifically, and by setting it as GPLv2 or later, the code will still be able to be integrated into GPLv3 projects of others.

We should also add additional copyright for @xantares and @samcv to the header of the file. I am willing to make these changes, but wanted to get confirmation that @xantares is okay with these changes, and I will go ahead and add a `LICENSE` file to the project and copyright information to the header. I will probably also update some of the header's outdated information while still referring to the original project our fork is based on.

**Comment by xantares** [comment]
yes, this is fine

**Comment by samcv** [comment]
Closed with c9548bd562a0b480868b886d1ef78801019b9b90
